.github/workflows/on-merge-to-main.yml

```diff
@@ -32,6 +40,9 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   release_label_on_merge:
+    permissions:
+      pull-requests: read
+      contents: write
```

Comment:
Can this `contents: write` be scoped down only to issues?

Reply:
With `issues: write` yes, but removing content would not prevent from merge, or?